webroot/adm/site.html

```diff
@@ -81,9 +81,13 @@ <h3>Output</h3>
         $('#doSetTypes').on('click', function () {
             var siteId = encodeURIComponent($('#siteId').val());
             var types = encodeURIComponent($('#types').val());
+            if(types.length < 1) {
+                $('#errorOutput').html('Can not set types with empty types');
```

Comment:
what if you accidentally changed an empty types to say DSP and you want to undo that?

Reply:
Perhaps we can use a `confirm` dialog? I know it adds to the 90s vibe of the website but we're kinda already there.
I imagine most of the time people not putting types is an error but occasionally it's intentional as Jonathan mentioned.

Reply:
Added a confirm with a message warning about what will happen